scripts/build

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+$(npm bin)/babel \
```

Comment:
Do you think it's better to keep the files separate or concatenate with webpack, as I do here: https://github.com/shakacode/node-webpack-async-example